Content.Server/Xenoarchaeology/Equipment/ArtifactAnalyzerSystem.cs

```diff
@@ -31,21 +39,59 @@ private void OnExtractButtonPressed(Entity<AnalysisConsoleComponent> ent, ref An
         if (!_research.TryGetClientServer(ent, out var server, out var serverComponent))
             return;
 
+        // Begin DeltaV - Variables for extraction totals
         var sumResearch = 0;
+        var sumGlimmer = 0;
+        // End DeltaV
+        ArtifactAnalyzerComponent? analyzer = null;
+        if (ent.Comp.AnalyzerEntity is { } analyzerNetEntity)
+        {
+            var analyzerEntityUid = GetEntity(analyzerNetEntity); // Convert NetEntity to EntityUid
+            TryComp<ArtifactAnalyzerComponent>(analyzerEntityUid, out analyzer);
+        }
```

Comment:
It looks like there's a TryGetAnalyzer function in SharedArtifactAnalyzerSystem. It has some additional checks. I'd recommend using that function here.